data/strings/strings.txt

```diff
@@ -21841,7 +21844,7 @@
 
   [keyboard_availability_alert]
     tags = ios
-    en = Keyboard is not available while driving
+    en = Keyboard is unavailable while driving
```

Comment:
I think "not available" is better.  unavailable has a connotation of meeting scheduling